HealthcareSystem/GraphicalEditor/Models/MapObjectRelated/MapObjectEntity.cs

```diff
@@ -12,6 +13,11 @@ public class MapObjectEntity : IdGenerator
         private MapObjectType _mapObjectType;
         private String _description;
 
+        public MapObjectEntity(string description = "")
+        {
+            _mapObjectType = new MapObjectType(MapObjectTypes.PARKING);
```

Comment:
Why is PARKING the default type?

Reply:
PARKING is the only map object that doesn't require any arguments to be created